src/test/java/com/microsoft/sqlserver/jdbc/unit/UTF8SupportTest.java

```diff
@@ -33,6 +34,7 @@
  * A class for testing the UTF8 support changes.
  */
 @RunWith(JUnitPlatform.class)
+@Tag("AzureTest")
 public class UTF8SupportTest extends AbstractTest {
     private static Connection connection;
     private static String databaseName;
```

Comment:
Do not tag this file for Azure testing as it creates Database during setup, add task to backlog.